src/views/uploadEvidence.html

```diff
@@ -0,0 +1,77 @@
+<main id="content" role="main">
+
+  {{> common/betaBanner }}
+
+  {{> common/backLink }}
+
+  <div class="grid-row">
+    <div class="column-two-thirds">
+
+      {{> common/errorSummary }}
+
+      <h1 id="page-heading" class="heading-large">{{ pageHeading }}</h1>
```

Comment:
This should be changed to use the new pageHeading partial.

Reply:
Done